api/v1alpha4/cluster_types.go

```diff
@@ -278,6 +278,7 @@ func (v APIEndpoint) String() string {
 // +kubebuilder:resource:path=clusters,shortName=cl,scope=Namespaced,categories=cluster-api
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
```

Comment:
following the example of other `additionalPrinterColumns` items, I think we want a `description="<insert description here"` for this new `Age` property

Reply:
fyi, the Age column of regular types has the following description:
```bash
CreationTimestamp is a timestamp representing the server time when this object was created. It is not guaranteed to be set in happens-before order across separate operations. Clients may not set this value. It is represented in RFC3339 form and is in UTC.\n\nPopulated by the system. Read-only. Null for lists. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Might be a bit long to use verbatim in our case

Reply:
Gotcha, I wrote the description as 'Time duration since creation of <resource_name>". How does that sound?

Reply:
Sounds good to me